rust/poem/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 poem = "~1.2.0"
-tokio = { version = "~1.15.0", features = ["rt-multi-thread", "macros"] }
+tokio = { version = "~1.16.0", features = ["rt-multi-thread", "macros"] }
```

Comment:
@sunli829 Is there a way not to specify version for `tokio` ? I mean put the version in `poem` itself